CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [Unreleased][Unreleased_log]
 --------------
+### Added
+- In Linux, 0-base enclave creation is enabled.
+  - Developers can create an enclave with base address 0x0 by either defining SGX_ENCLAVE_ZERO_BASE (before including headers) or
```

Comment:
Can we avoid requiring that the user define SGX_ENCLAVE_ZERO_BASE?
There is no other feature, that I can think of, that uses conditional compilation.